packages/core/admin/admin/src/hooks/useRolesList/index.js

```diff
@@ -26,7 +27,7 @@ const useRolesList = (shouldFetchData = true) => {
 
       const {
         data: { data },
-      } = await axiosInstance.get('/admin/roles');
+      } = await fetchClient.get('/admin/roles');
```

Comment:
Same here

Reply:
I needed the fetchClient variable to execute the get method because we also imported the lodash get method